feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt

```diff
@@ -29,6 +29,9 @@ fun RepairView() {
     var isAValidDescription by remember {
         mutableStateOf(false)
     }
+    var isAValidNumber by remember {
```

Comment:
You can declare the variable like `isValidPhone`

Reply:
Ok, will change it